api/audit/models.py

```diff
@@ -18,6 +18,10 @@
 FEATURE_STATE_UPDATED_MESSAGE = (
     "Flag state / Remote Config value updated for feature: %s"
 )
+FEATURE_STATE_WENT_LIVE_MESSAGE = (
+    "Flag state / Remote Config value went live for feature: %s"
```

Comment:
Maybe we should include the term 'scheduled' in here somewhere?

```
"Scheduled change to Flag state / Remote config value went live for feature: %s"
```

Is there a way that we can include the change request information too?

Reply:
Yes, I have updated it to include the title